__test__/operatives/[operativeId]/work-orders/[id]/close.test.js

```diff
@@ -0,0 +1,26 @@
+import {
+  AGENT_ROLE,
+  AUTHORISATION_MANAGER_ROLE,
+  CONTRACTOR_ROLE,
+  CONTRACT_MANAGER_ROLE,
+  OPERATIVE_ROLE,
+} from 'src/utils/user'
+import OperativeWorkOrderClosePage from '../../../../../src/pages/operatives/[operativeId]/work-orders/[id]/close'
```

Comment:
I think Evangelos added some path aliases the other day - maybe add one for this though you could leave it to another PR